db/models/Word.js

```diff
@@ -0,0 +1,261 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+const wordSchema = new Schema({
+  user: { type: String },
+  category: { type: String, required: true },
+  base: {
+    language: { type: String, required: true },
+    flag: { type: String, required: true },
+    translation: { type: String, required: true },
+  },
+  notes: { type: String },
+  query1: {
+    language: { type: String, required: true },
+    flag: { type: String, required: true },
+    translation: { type: String, required: true },
+    declension: {
+      specification: { type: String },
+      singular: {
+        nominative: { type: String },
+        genitive: { type: String },
+        dative: { type: String },
+        accusative: { type: String },
+        locative: { type: String },
+        instrumental: { type: String },
+      },
+      plural: {
+        nominative: { type: String },
+        genitive: { type: String },
+        dative: { type: String },
+        accusative: { type: String },
+        locative: { type: String },
+        instrumental: { type: String },
+      },
+    },
+    conjugation: {
+      present: [
+        {
```

Comment:
Here comes a lot of repetition. Instead you could declare a new Schema in the beginning of this file like so:

```javascript
const conjugationSchema = new Schema({ 
person: {type: String}, 
pronouns: {type:String}, 
verbForm: {type: String}})
```

and apply it to your conjugation like so: 

```javascript
conjugation: {
present: [conjugationSchema],
past: [conjugationSchema],
...}
```
This should work.